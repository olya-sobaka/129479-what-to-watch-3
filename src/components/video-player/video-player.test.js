import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.js";

const movie = {
  title: `Kiki's Delivery Service`,
  poster: `img/midnight-special.jpg`,
  id: 36,
  previewMp4: `https://coverr.co/videos/norway-road-f0J1etTrcQ`,
  previewWebm: `https://coverr.co/videos/morning-sunshine-Of9KYVXN1C`
};

it(`VideoPlayer renders correctly`, () => {
  const tree = renderer.create(
      <VideoPlayer film={movie}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
