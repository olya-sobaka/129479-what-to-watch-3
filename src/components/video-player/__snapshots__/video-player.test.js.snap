// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer renders correctly 1`] = `
<video
  className="player__video"
  height="175"
  muted={true}
  poster="img/midnight-special.jpg"
  width="280"
>
  <source
    src="https://coverr.co/videos/norway-road-f0J1etTrcQ"
    type="video/mp4"
  />
  <source
    src="https://coverr.co/videos/morning-sunshine-Of9KYVXN1C"
    type="video/webm"
  />
</video>
`;
